gutenberg/acquire/text.py

```diff
@@ -57,10 +74,10 @@ def _format_download_uri(etextno):
         response = requests.head(uri)
         if response.ok:
             return uri
-    raise UnknownDownloadUriException
+    raise UnknownDownloadUriException("Failed to find {} on {}.".format(etextno, uri_root))
```

Comment:
Nitpick: elsewhere we add the index to the format strings, i.e. `{0}` over `{}`.